lib/rust/ensogl/pack/js/src/runner/index.ts

```diff
@@ -232,8 +229,12 @@ export class App {
     }
 
     /** Log the message on the remote server. */
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     remoteLog(message: string, data: any) {
-        // TODO: Implement remote logging. This should be done after cloud integration.
+        // FIXME [PB]: https://github.com/enso-org/cloud-v2/issues/359
```

Comment:
Added to cloud board since Elasticsearch integration is on cloud side. This way we will be remember to integrate code here